Sources/bitrise-step-swift-clone/Config.swift

```diff
@@ -0,0 +1,46 @@
+import Foundation
+
+struct Config {
+    let repositoryURL: String
+    let cloneDestination: String
+    let checkoutBranch: String?
```

Comment:
I wonder if these should be encapsulated into a inner struct called Options for checkoutBranch and clone depth.  Also maybe there is a real life example to draw from